Assets/Resources/ScriptableObjects/ItemSO.cs

```diff
@@ -0,0 +1,19 @@
+using UnityEngine;
+
+[CreateAssetMenu(menuName = "ScriptableObjects/Item")]
+public class ItemSO : ScriptableObject
+{
+    public string ItemName;
+    public float ProbabilityPercentage;
+    public Sprite ItemSprite;
+
+    [System.Serializable]
+    public class StatModifier
+    {
+        public StatType Stat;
+        public float Value;
```

Comment:
Ok, but some changes may be non-permanent boosters, shields, etc. or this just for permanent things and skills also will be stats?

Reply:
For now, only permanent stat changing.